timemachine/fe/free_energy.py

```diff
@@ -245,7 +245,7 @@ def _combine(self, ligand_values, host_values=None):
 
 # this class is serializable.
 class RelativeFreeEnergy(BaseFreeEnergy):
-    def __init__(self, top: Union[topology.SingleTopology, topology.DualTopology], label=None, complex_path=None):
+    def __init__(self, top: topology.DualTopology, label=None, complex_path=None):
```

Comment:
It looks like there's still code in this class to deal with `SingleTopology` (L345). Can that be removed, or should we continue to accept `Union[SingleTopology, DualTopology]` here?